### ISSUE TITLE
Multiple HA deployment modes

### ISSUE THREAD

**Issue by wanghangyu817** [zendesk/maxwell#1946]
maxwell is currently highly available through jgroups, which is a good choice.  However, in many cases, users are not familiar with jgroups.  On the contrary, users who deploy maxwell in a big data architecture will be more familiar with zookeeper.  Whether maxwell can co-exist with multiple HA deployment modes?  Users can choose to use the distributed coordination technology of the underlying high availability.  If so, I will submit maxwellHA on ZK

**Comment by osheroff** [comment]
sure, if you like.  Honestly kubernetes is a much better answer here but hey

**Comment by wanghangyu817** [comment]
Yes, maxwell is more flexible by providing users with a variety of options
